src/opening-period/time-span/Weekdays.tsx

```diff
@@ -1,29 +1,34 @@
 import React from 'react';
 import './Weekdays.scss';
+import { ArrayField } from 'react-hook-form';
+import { TimeSpanFormFormat, FormWeekdays } from '../../common/lib/types';
 
 const DayCheckbox = ({
   register,
   dataTest,
   timeSpanIndex,
   dayIndex,
+  weekdays = [],
```

Comment:
I wonder if we could replace this with defaultChecked value? Im not sure if the component needs the whole weekdays array?

Reply:
It did not work when I tested it the last time. I may have not casted it then but I do not see the problem since dayIndex is passed also

Reply:
Ouh I see. If this is the way it works then let's leave it like it is.

Reply:
I can also pass the value